css/mystyle.css

```diff
@@ -142,6 +141,9 @@ margin-top: 1.563rem;
     margin-top: 2em;
     letter-spacing: 0.03em
 }
+section.goals p {
+	padding: 0 15px;
```

Comment:
Added some padding to the paragraphs of text so that when the screen gets smaller there's some space around the text.